installation.md

```diff
@@ -87,10 +87,14 @@ should work with any of them, though feature set and interface may vary.
 Here are some plugins we know work well with clangd:
 
 <details>
-<summary markdown="span">Vim</summary>
-[YouCompleteMe](https://ycm-core.github.io/YouCompleteMe/) can be installed with
-clangd support. **This is not on by default**, you must install it with
-`install.py --clangd-completer`.
+<summary markdown="span">Vim/Neovim</summary>
+clangd runs through [Language Server Protocol](https://microsoft.github.io/language-server-protocol/),
```

Comment:
This line, "clangd runs through the Language Server Protocol", is applicable to all editors. Let's move it up to the beginning of the "Editor plugins" section.